test/integration_test_helper.rb

```diff
@@ -20,18 +20,63 @@
 end
 
 Selenium::WebDriver::Chrome::Service.driver_path = ENV['TESTDRIVER_PATH'] || Foreman::Util.which('chromedriver', Rails.root.join('node_modules', '.bin'))
-Capybara.register_driver :selenium_chrome do |app|
+
+javascript_driver = ENV.fetch("JS_TEST_DRIVER") { ENV['DEBUG_JS_TEST'] ? :selenium_chrome : :selenium_chrome_headless }.to_sym
+
+def chrome_options
   options = Selenium::WebDriver::Chrome::Options.new
-  options.args << '--disable-gpu'
-  options.args << '--no-sandbox'
   options.args << '--window-size=1024,768'
-  options.args << '--headless' unless ENV['DEBUG_JS_TEST'] == '1'
-  Capybara::Selenium::Driver.new(app, browser: :chrome, options: options)
+  options.args += ENV.fetch('ADDITIONAL_CHROME_OPTIONS', '').split(';')
+  options
+end
+
+if javascript_driver == :selenium_chrome_remote
+  ShowMeTheCookies.register_adapter(:selenium_chrome_remote, ShowMeTheCookies::SeleniumChrome)
+
+  Capybara.register_driver :selenium_chrome_remote do |app|
+    selenium_remote_host = ENV.fetch('SELENIUM_REMOTE_HOST')
+    selenium_remote_port = ENV.fetch('SELENIUM_REMOTE_PORT', 4444)
+    Capybara::Selenium::Driver.new(
+      app,
+      browser: :remote,
+      url: "http://#{selenium_remote_host}:#{selenium_remote_port}/wd/hub",
+      options: chrome_options)
+  end
+elsif javascript_driver == :selenium_chrome_headless
+  options = chrome_options
+  options.args << '--headless'
+  Capybara.register_driver :selenium_chrome_headless do |app|
+    Capybara::Selenium::Driver.new(
+      app,
+      browser: :chrome,
+      options: options)
+  end
```

Comment:
Why is this needed? This should be available by default:
https://github.com/teamcapybara/capybara/blob/0480f90168a40780d1398c75031a255c1819dce8/lib/capybara/registrations/drivers.rb#L31-L42

I had hoped to utilize this if possible. Given we don't set the window size elsewhere, I'd be tempted to drop `chrome_options`. I'm not sure anyone actually uses `ADDITIONAL_CHROME_OPTIONS` anyway.

Reply:
As far as I remember (I touched this code 5 months ago) but the window size was required. Although the discussion if that is needed is IMO out of scope for this PR (can be done in follow up PR)

I'm requiring `ADDITIONAL_CHROME_OPTIONS` downstream. This will make it more easily configurable.

Reply:
I think its a good idea to be able to set additonal options which can be used. They are optional. So, if you don't need them, don't set the option and it doesn't hurt.